magni_gazebo/scripts/motion_vectors.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+
+import rospy
+from raspicam_node.msg import MotionVectors
```

Comment:
It would be a nice-to-have to have the raspicam_node dependency listed in package.xml so the rosdep can take care of this :)